docs/reference/scripting/security.asciidoc

```diff
@@ -61,3 +67,42 @@ For example, to allow scripts to run only in `scoring` and `update` contexts:
 ----
 script.allowed_contexts: score, update
 ----
+
+[[allowed-script-in-aggs-settings]]
+[discrete]
+=== Allowed scripts in scripted metrics aggregations
+
+By default, all scripts are permitted in
+<<search-aggregations-metrics-scripted-metric-aggregation,scripted metrics aggregations>>.
+To restrict the set of allowed scripts, set
+<<search-settings-only-allowed-scripts,`search.aggs.only_allowed_metric_scripts`>>
+to `true` and provide the allowed scripts using
+<<search-settings-allowed-inline-scripts,`search.aggs.allowed_inline_metric_scripts`>>
+and
```

Comment:
```suggestion
and/or
```

nit for clarity, you can provide scripts either inline or stored, or both IIUC?

Reply:
Both! Added a sentence to clarify this.